### ISSUE TITLE
Economy

### ISSUE THREAD

**Issue by cmcintosh** [cmcintosh/VillagerLife#49]
The economy of the mod will be fleshed out with the use of Emeralds to purchase buildings, supplies, and trades between villages.

All building tokens will have various prices, that will fluctuate based on production/availability and recent purchase histories, this will also be applied to other items. See https://www.peachpit.com/articles/article.aspx?p=1925649&seqNum=2 as a reference.

**Comment by 1yeetus2** [comment]
Good idea, this is a good addition to the current economy of the tradesman and architect
